server/src/main/kotlin/core/gameflow/Blinds.kt

```diff
@@ -1,3 +1,3 @@
 package core.gameflow
 
-data class Blinds(val bigBlind: Int, val smallBlind: Int, val ante: Int = 0)
+data class Blinds(val smallBlind: Int, val bigBlind: Int, val ante: Int = 0)
```

Comment:
ocd driven develpement(ODD)

Reply:
I found it more convenient when writing tests to put SB before BB (when describing blinds in a poker game SB is always first, e.g. 100$/200$ game, so it is more readable to do the same in the code).

Reply:
thanks for the explanation

Reply:
you're welcome :)